tools/mtouch/Errors.designer.cs

```diff
@@ -30,7 +30,7 @@ internal class Errors {
         internal static System.Resources.ResourceManager ResourceManager {
             get {
                 if (object.Equals(null, resourceMan)) {
-                    System.Resources.ResourceManager temp = new System.Resources.ResourceManager("Errors.mtouch", typeof(Errors).Assembly);
+                    System.Resources.ResourceManager temp = new System.Resources.ResourceManager("mtouch.Errors", typeof(Errors).Assembly);
```

Comment:
I need to figure out why this change occurred. I believe this is causing a name mismatch since mtouch, mmp, and dotnet-linker are looking for the resources named "Errors.mtouch" and not "mtouch.Errors"